### PR TITLE
getStatusCode() support

### PR DESCRIPTION
**Pull request by dkarlovi**
Adds support for Mink's `getStatusCode()`

### CODE REVIEW

**Comment by dkarlovi** [comment]
Build failures unrelated, don't know what's that about.

File: src/PantherDriver.php [hunk 1]
```diff
@@ -39,6 +39,9 @@
 class PantherDriver extends CoreDriver
 {
     use PantherTestCaseTrait;
+    
+    private const CHROME = 'chrome';
+    private const FIREFOX = 'firefox';
```
Comment (robertfausk) [inline src/PantherDriver.php:44]: You can remove these lines. They are already merged and not relevant to this PR.

**Review by robertfausk — CHANGES_REQUESTED** [review]
This library currently does not have fully support for symfony/panther with HttpBrowserClient atm.
It's unlikely that I find time to add tested support for HttpBrowserClient in the near future but PRs are always welcome!:)

We also need some tests for different status codes.

File: src/PantherDriver.php [hunk 2]
```diff
@@ -164,6 +167,14 @@ public function visit($url)
         $this->client->get($this->prepareUrl($url));
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getStatusCode()
+    {
+        return $this->client->getInternalResponse()->getStatusCode();
```
Comment (robertfausk) [inline src/PantherDriver.php:175]: Here we need to distinguish between the types of the client: PantherClient and HttpBrowserClient
The code would be similar to the following:
```PHP
public function getStatusCode()
{
    if ($this->client instanceof HttpBrowserClient) {
        return $this->client->getInternalResponse()->getStatusCode();
    }
    throw new UnsupportedDriverActionException('Status code is not available from "%s" with client of type "%s"', $this, \get_class($this->client));
}
```

**Pull request closed without merging**